packages/InlineEditors/src/Editor/Editor.js

```diff
@@ -33,15 +92,29 @@ Editor.Edit = () => {
 };
 
 Editor.Edit.displayName = "Editor.Edit";
+Editor.types = { status: statusTypes };
 
 const propTypes = {
+  status: PropTypes.oneOf([
+    Editor.types.status.ERROR,
+    Editor.types.status.IDLE,
+    Editor.types.status.LOADING,
+    Editor.types.status.SUCCESS,
+  ]),
   children: PropTypes.node.isRequired,
   isEditing: PropTypes.bool.isRequired,
+  onAnimationEndSuccess: PropTypes.func,
   onClick: PropTypes.func,
+  optimisticValue: PropTypes.node,
+  messageError: PropTypes.node,
 };
 
 const defaultProps = {
+  onAnimationEndSuccess: () => {},
```

Comment:
I think this name is a little bit confusing, what about `onSuccessAnimationEnd`? why we need this callback? there's something that has to happen after the green checkmark disappears?

Reply:
yes! we have to revert the internal state into IDLE state again so is possible to edit again the ListBox (or any other element) and not show the Success (check icon) :)

Reply:
I will change the name